crates/cli-flags/src/lib.rs

```diff
@@ -42,20 +42,18 @@ wasmtime_option_group! {
         /// Optimization level of generated code (0-2, s; default: 2)
         pub opt_level: Option<wasmtime::OptLevel>,
 
-        /// Force using a "static" style for all wasm memories
-        pub static_memory_forced: Option<bool>,
+        /// Do not allow memories to grow beyond `-O memory-reservation`
+        pub memory_may_move: Option<bool>,
```

Comment:
This should be reworded, maybe

```suggestion
        /// Do not allow Wasm linear memories to move in the host
        /// process's address space.
        pub memory_may_move: Option<bool>,
```